backends/azure_table_test.go

```diff
@@ -1,16 +1,19 @@
 package backends
 
 import (
-	"github.com/satori/go.uuid"
+	"github.com/prebid/prebid-cache/utils"
 	"testing"
 )
 
 func TestPartitionKey(t *testing.T) {
 	azureTable := NewAzureBackend("abc", "def")
 
-	id := uuid.NewV4().String()
-	expected := id[0:4]
+	id, err := utils.GenerateUUIDString()
+	if err != nil {
+		t.Errorf("Error generating version 4 UUID")
+	}
 
+	expected := id[0:4]
```

Comment:
It looks like we only ever use the first 4 characters, consider including the shortening in `GenerateUUIDString` and rename it to `GenerateRandomID`.

Reply:
I don't think we should make `GenerateUUIDString()` to only return a 4 character string, `backends/aerospike.go`'s `Put` function uses the entire string. Did I misunderstand something? Do you mean create a function wrapper for `GenerateUUIDString()` only in `backends/azure_table_test.go` and make it return the first 4 characters only?
```
 func (a *Aerospike) Put(ctx context.Context, key string, value string, ttlSeconds int) error {
     asKey, err := as.NewKey(a.cfg.Namespace, setName, key)  // <-- gets passed the entire key
     if err != nil {
         return err
     }
     if ttlSeconds == 0 {
         ttlSeconds = a.cfg.DefaultTTL
     }
     bins := as.BinMap{
         binValue: value,
     }
     err = a.client.Put(&as.WritePolicy{
         Expiration: uint32(ttlSeconds),
     }, asKey, bins)
     if err != nil {
         return err
     }
     return nil
 }
 backends/aerospike.go
```

Reply:
Done. `GenerateUUIDString` has been renamed to `GenerateRandomID`